tests/Aspire.Dashboard.Tests/Integration/IntegrationTestHelpers.cs

```diff
@@ -29,8 +28,11 @@ public static class IntegrationTestHelpers
         {
             [DashboardWebApplication.DashboardUrlVariableName] = "http://127.0.0.1:0",
             [DashboardWebApplication.DashboardOtlpUrlVariableName] = "http://127.0.0.1:0",
-            [DashboardWebApplication.DashboardOtlpAuthModeVariableName] = nameof(OtlpAuthMode.None)
+            [DashboardWebApplication.OtlpAuthModeKey] = "Unsecured",
```

Comment:
Use name of instead of strings, e.g. `nameof(OtlpAuthMode.Unsecured)`

Reply:
Enum is not public.